ci_framework/roles/dlrn_promote/tasks/check_for_previous_promotions.yml

```diff
@@ -0,0 +1,39 @@
+---
+- name: Check reports from DLRN
+  ansible.builtin.shell:
+    cmd: >-
+      set -o pipefail;
+      export SSL_CA_BUNDLE="/etc/pki/ca-trust/extracted/openssl/ca-bundle.trust.crt";
```

Comment:
is that for d/stream only? will we ever need to override that? If we do we should consider adding a 'ssl_ca_bundle' ansible var into the role defaults

Reply:
Thank you @marios for the review. It will be used for downstream and upstream both. I have added a new var in the updated pr.